docs/core/introduction.md

```diff
@@ -139,6 +139,13 @@ The SDK download includes the following components:
 * The ASP.NET Core runtime. Provides basic services for internet-connected apps, such as web apps, IoT apps, and mobile backends.
 * The desktop runtime. Provides basic services for Windows desktop apps, including Windows Forms and WPF.
 
+The runtime download includes the following components:
+
+* The desktop or ASP.NET Core runtime.
```

Comment:
```suggestion
* Optionally, the desktop or ASP.NET Core runtime.
```

Reply:
As a part of .NET runtime download, we need to download either ASP.NET Core runtime or desktop runtime. The first point is mandatory. So why we should make the first point as optional?

https://dotnet.microsoft.com/download/dotnet/current/runtime

Reply:
See this page:  https://dotnet.microsoft.com/download/dotnet/5.0

Reply:
Thanks for the information. I have made the suggested changes.